### ISSUE TITLE
UTF-8 crash while using the HTTP request example.

### ISSUE THREAD

**Issue by TomatoCo** [KennyShields/LoveFrames-demo#4]
Running the demo, hitting Show Examples, scrolling down to the HTTP request example, sending the default request, and then scrolling all the way up, and then up once more, causes a crash.

For what it's worth, the text box starts with a square unrecognized character.

http://i.imgur.com/IgA8iAJ.png is the error.

http://i.imgur.com/g67u4GL.png is the state that the demo was in just prior to the crash. However, even just alt-tabbing back in to the demo from capturing the screenshot triggered a very similar crash, http://i.imgur.com/eBdLXhb.png , not just scrolling up.

**Comment by KennyShields** [comment]
Yeah, Love Frames doesn't handle utf8 text very well in its current state and is prone to issues like these. Might just change the default request url for now. Thanks for the report.
